packages/sync-server/src/app-sync.ts

```diff
@@ -372,9 +372,7 @@ app.get('/get-user-file-info', (req, res) => {
 });
 
 app.post('/delete-user-file', (req, res) => {
-  const { fileId } = req.body;
```

Comment:
This line was failing as req.body was undefined, which Express caught and threw a `500` instead of the desired `422`.

Reply:
I've cleaned up the others in https://github.com/actualbudget/actual/pull/5042/commits/2a1a7d7a234f473eec4453c3c0b51726f2ef8d3d

Expanding the sync-server test suite has been on my to do list for a little while. Maybe after this is a good time to do it.